rust_dev_preview/README.md

```diff
@@ -18,6 +18,37 @@ These examples demonstrate how to perform several operations using the developer
 
 ## Running the code examples
 
+### Environment Variables
+
+Unless otherwise noted, the AWS SDK for Rust uses these same environment variables as the AWS Command Line Interface (AWS CLI) to load account credentials, specify the default region, and to set other configuration settings.
```

Comment:
I think the SDK supports far fewer env vars than the CLI, and this isn't likely to change.

Reply:
I was trying to say that, for the following env vars, they intend to have the same meaning as their CLI counterpart. Does this work better?